src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -337,4 +338,20 @@ public static ProjectListOption fields(ProjectField... fields) {
    *     Platform Services</a>
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
+
+  /**
+   * Gets the latest state of a long-running operation.
+   *
+   * <p>Clients can use this method to poll the operation result at intervals as recommended by the
+   * API service.
+   *
+   * @param name The name of the operation resource.
+   * @return This resource represents a long-running operation that is the result of a network API
+   *     call.
+   * @throws ResourceManagerException upon failure
+   * @see <a href=
+   *     "https://cloud.google.com/resource-manager/reference/rest/v1/operations/get">Resource
+   *     Manager get</a>
+   */
+  Operation getOperations(String name);
```

Comment:
Method is name getOperation**s** , but returned object is Operation.
Is it what you expect?

> @return This resource represents a long-running operation that is the result of a network API call.

It's not clear to me from the sentence what is returning. I guess it should be like
@return The result of a long-running operation

Reply:
done